packages/runtime/src/components/page/Page.tsx

```diff
@@ -282,9 +282,9 @@ export function Page({ page, preview = false }: Props): JSX.Element {
         {socialImage && (
```

Comment:
LGTM! 

Curious how you realized that this was a problem. Were you just sharing on twitter?

Reply:
Andrew created an issue for it last week: https://linear.app/makeswift/issue/PRD-402/preview-card-isnt-displaying-in-twitter-when-sharing-a-makeswift-site